protocol/x/clob/memclob/memclob.go

```diff
@@ -153,6 +153,7 @@ func (m *MemClobPriceTimePriority) CountSubaccountOrders(
 	subaccountId satypes.SubaccountId,
 	filter func(types.OrderId) bool,
 ) (count uint32) {
+	lib.AssertCheckTxMode(ctx)
```

Comment:
Suggestion: can we update this function to just count Short-Term orders? It seems like that's all it's used for now.

I don't think this function should ever be used outside of that context.

Also update the function comment to note this function should only be called in `CheckTx` / `PrepareCheckState` and is inherently non-deterministic.

Reply:
I updated the comment and updated this method and the `CountUntriggeredSubaccountOrders` as well.